hapi-fees/src/main/java/com/hedera/services/usage/token/entities/TokenEntitySizes.java

```diff
@@ -55,8 +54,8 @@ public int bytesUsedToRecordTokenTransfers(int numTokens, int fungibleNumTransfe
 				+ uniqueNumTransfers * USAGE_PROPERTIES.nftTransferBytes();
 	}
 
-	public long bytesUsedForUniqueTokenTransfers(int numTokens) {
-		return numTokens * (BASIC_RICH_INSTANT_SIZE + BASIC_ENTITY_ID_SIZE + LONG_SIZE + BASIC_ENTITY_ID_SIZE);
+	public long bytesUsedForUniqueTokenTransfers(int numOwnershipChanges) {
+		return numOwnershipChanges * (BASIC_ENTITY_ID_SIZE + LONG_SIZE + BASIC_ENTITY_ID_SIZE);
```

Comment:
This could be reduced further to `return numOwnershipChanges * (2 * BASIC_ENTITY_ID_SIZE + LONG_SIZE);`